src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir" ;
+  font-weight: 300;
+  font-style: normal;
+  src: local("Avenir"), url(/fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "AvenirBook" ;
+  font-weight: 300;
+  font-style: normal;
+  src: local("Avenir-Book"), url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "AvenirHeavy" sans-serif;
+  font-weight: 900;
+  font-style: normal;
+  src: local("Avenir-Heavy"), url(/fonts\Avenir-Heavy.ttf);
+}
+
+body {
+  box-sizing: border-box ;
+  margin: 0 auto;
+  padding: 0 9px;
+  border: 0;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.container {
+  width: 100% ;
+  display: flex;
+  flex-direction: column ;
+  align-items: center ;
+  justify-content: center ;
+}
+
+.search_form {
+  width: 100%;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.form_input {
+  position: relative ;
+  display: flex;
+  align-items: center ;
+  justify-content: center ;
+  width: 100% ;
+  color: #3d4e61 ;
+}
+
+.big_input input {
+  width: 100% ;
+  padding: 23px 10px 23px 62px ;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 900;
+  border: none;
+  outline: none;
```

Comment:
Default input styles don't have the outline, so you don't need to remove it.

![image](https://user-images.githubusercontent.com/56752562/220862350-cc8650bb-9692-48f6-96f9-4b19bd9f0c8f.png)